package.json

```
@@ -61,7 +61,7 @@
     "build-docker-images": "tsx ./docker/buildDocker.ts",
     "test-mongo": "npm run build && npm run build-tests && MONGO_TESTS='true' DISABLE_PLUGINS='true' node --no-warnings --experimental-require-module --no-experimental-strip-types --test",
     "test": "npm run build && npm run build-tests && cross-env DISABLE_PLUGINS='true' node --no-warnings --experimental-require-module --no-experimental-strip-types --test",
-    "test-dev": "npm run lint -- --quiet; npm run check_prettier; npm t",
+    "test-dev": "npm run lint --quiet; npm run check_prettier; npm t",
```

```suggestion
    "test-dev": "npm run lint -- --quiet; npm run check_prettier; npm t",
```
The first "--" tell npm to apply the flag to the inner command it wasn't a fail